.github/workflows/build.yml

```
@@ -0,0 +1,33 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    env:
+      WHITEPAPER_FORMATS_DIR: whitepaper/formats
+      WHITEPAPER_FILE_NAME_ROOT: gridcoin-whitepaper-draft
+      WHITEPAPER_ARTIFACT_NAME: gridcoin-whitepaper
+    steps:
+    - uses: actions/checkout@v2
```

```suggestion
    - uses: actions/checkout@v3
```
Could you update this to v3? v2 leaves warnings in the build logs as it uses some deprecated method.